test/util/fullNet/helpers.ts

```diff
@@ -259,7 +259,7 @@ export const runSimpleFullNet = async (bspNetConfig: BspNetConfig) => {
           [multiAddressMsp1],
           1,
           "Terms of Service...",
-          500,
+          9999999,
```

Comment:
it's an arbitrarily large value set to the value proposition bucket size limit to avoid all tests hitting an upper bound bucket size when issuing storage requests.